picard/file.py

```diff
@@ -158,6 +158,28 @@ def _load(self, filename):
         """Load metadata from the file."""
         raise NotImplementedError
 
+    def _test_same(self):
+        if config.setting['clear_existing_tags']:
+            return False
+        new_metadata = self.new_metadata
+        orig_metadata = self.orig_metadata
+        tags = set(new_metadata.keys() + orig_metadata.keys())
+        for name in filter(lambda x: x.startswith("~config:"), tags):
+            new_metadata[name] = config.setting[name.split(':')[1]]
+            if new_metadata[name] != orig_metadata[name]:
+                return False
+        for name in filter(lambda x: (not x.startswith("~")
+                                      or x.startswith("~id3")
+                                      or x == "~rating")
+                                    and self.supports_tag(x), tags):
+            new_values = new_metadata.getall(name)
+            orig_values = orig_metadata.getall(name)
+            if new_values != orig_values:
+                return False
+        if abs(float(orig_metadata.length) - float(new_metadata.length)) > 2000:
```

Comment:
2000 here (and in the tagdiff code) should be moved to a new constant in `picard/const/__init__.py`, like `MAX_INSIGNIFICANT_LENGTH_DIFFERENCE` or something like that.